platforms/ide/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/TestExecutionResultEvaluator.java

```diff
@@ -132,35 +131,40 @@ private String formatInternalTestExecutionRequest() {
             final String className = internalJvmTestRequest.getClassName();
             final String methodName = internalJvmTestRequest.getMethodName();
             if (methodName == null) {
-                requestDetails.append("\n").append(Strings.repeat(INDENT, 2)).append("Test class ").append(className);
+                requestDetails.append("\n").append(twoIntent()).append("Test class ").append(className);
             } else {
-                requestDetails.append("\n").append(Strings.repeat(INDENT, 2)).append("Test method ").append(className).append(".").append(methodName).append("()");
+                requestDetails.append("\n").append(twoIntent()).append("Test method ").append(className).append(".").append(methodName).append("()");
             }
         }
 
         for (InternalTaskSpec taskSpec : internalTestExecutionRequest.getTaskSpecs()) {
             if (taskSpec instanceof InternalTestSpec) {
                 InternalTestSpec testSpec = (InternalTestSpec) taskSpec;
                 for (String cls : testSpec.getClasses()) {
-                    requestDetails.append("\n").append(Strings.repeat(INDENT, 2)).append("Test class: ").append(cls).append(" in task " + taskSpec.getTaskPath());
+                    requestDetails.append("\n").append(twoIntent()).append("Test class: ").append(cls).append(" in task " + taskSpec.getTaskPath());
                 }
                 for (Map.Entry<String, List<String>> methods : testSpec.getMethods().entrySet()) {
                     for (String method : methods.getValue()) {
-                        requestDetails.append("\n").append(Strings.repeat(INDENT, 2)).append("Test method ").append(methods.getKey()).append(".").append(method).append("()").append(" in task " + taskSpec.getTaskPath());
+                        requestDetails.append("\n").append(twoIntent()).append("Test method ").append(methods.getKey()).append(".").append(method).append("()").append(" in task " + taskSpec.getTaskPath());
                     }
                 }
                 for (String pkg : testSpec.getPackages()) {
-                    requestDetails.append("\n").append(Strings.repeat(INDENT, 2)).append("Test package ").append(pkg).append(" in task " + taskSpec.getTaskPath());
+                    requestDetails.append("\n").append(twoIntent()).append("Test package ").append(pkg).append(" in task " + taskSpec.getTaskPath());
                 }
                 for (String pattern : testSpec.getPatterns()) {
-                    requestDetails.append("\n").append(Strings.repeat(INDENT, 2)).append("Test pattern ").append(pattern).append(" in task " + taskSpec.getTaskPath());
+                    requestDetails.append("\n").append(twoIntent()).append("Test pattern ").append(pattern).append(" in task " + taskSpec.getTaskPath());
                 }
             }
         }
 
         return requestDetails.toString();
     }
 
+    @SuppressWarnings("InlineMeInliner")
+    private static String twoIntent() {
```

Comment:
s/Intent/Indent/

Reply:
Maybe take a parameter (`indent(2)` is at least as clear as `twoIndent()` IMO)

Reply:
> s/Intent/Indent/

good catch

Reply:
> Maybe take a parameter (`indent(2)` is at least as clear as `twoIndent()` IMO)

It would complain about always having the same parameter :)